src/app/frontend/deploy/deployfromsettings_controller.js

```diff
@@ -232,4 +235,17 @@ export default class DeployFromSettingsController {
    * @private
    */
   getName_() { return this.name; }
+
+  /**
+   * Returns true if more options have been enabled and should be shown, false otherwise.
+   * @return {boolean}
+   * @export
+   */
+  isMoreOptionsEnabled() { return this.showMoreOptions_; }
+
+  /**
+   * Shows or hides more options.
+   * @export
+   */
+  switchMoreOptions() { this.detail.showMoreOptions_ = !this.detail.showMoreOptions_; }
```

Comment:
Awesome. I like that you didn't put this logic into the template :)

Can we have a test for this?